### DISCUSSION TITLE
2007/10/29/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83-%E6%A2%A6%E4%B8%AD%E7%9A%84%E9%A3%8E

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#186] [Announcements]
# 2007/10/29/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83-%E6%A2%A6%E4%B8%AD%E7%9A%84%E9%A3%8E

我站在大楼的窗前向外张望，突然间就看到一个接一个的黑色风柱夹杂着砂石呼啸而来。

https://blog.qizhen.xyz/2007/10/29/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83-%E6%A2%A6%E4%B8%AD%E7%9A%84%E9%A3%8E

**Comment by ruanqizhen** [comment]
之前博客下的留言：

arrowpig
2007年10月30日 19:22 编辑
有没有分析的阿！？

回复
David
2007年10月29日 09:33 编辑
恭喜你成为Heros一员，Save the cheerleader, save the world！

回复
千纤草
2007年10月29日 09:00 编辑
有人说，做梦是对现实的不满，不晓得是不是真的，反正我也做很多梦~

回复
